release_automation.sh

```diff
@@ -271,6 +271,9 @@ if [[ $REPLY =~ ^[Nn]$ ]]; then
     read -r -p "Enter the branch name you want to target. Make sure a branch with this name already exists in WPiOS repository: " WPIOS_TARGET_BRANCH
 fi
 
+ohai "Update the spec repos located at ~/.cocoapods/repos"
```

Comment:
Would it be better to group this with the other commands starting at line 295? Let me know the rationale for its placement here. Thanks!

Reply:
You're right that its placement here doesn't make much sense.
I'm going to move it to the top of the script because it's a command that update all CocoaPod specs, so it's not specific to WPiOS (it also affects Gutenberg).
 I'm going to move it up to the top of the script so that it's one of the first things done and then all other repos that have pods (Gutenberg and WPiOS) can benefit from it.